sdk/ai/Azure.AI.Projects/src/Custom/Agent/OpenApiToolDefinition.cs

```diff
@@ -17,6 +17,7 @@ public OpenApiToolDefinition(string name, string description, BinaryData spec, O
                     description: description,
                     spec: spec,
                     auth: auth,
+                    defaultParams: [],
```

Comment:
Nikolay mentioned defaultParams shall be in OpenApiToolDefinition as well?

Reply:
Fixed.